src/Money.php

```diff
@@ -515,6 +515,66 @@ public function jsonSerialize()
         ];
     }
 
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
+     *
+     * @return Money
+     */
+    public static function minimum(self $first, self ...$collection)
```

Comment:
I believe max/min/avg are well-known forms aggregation functions. Sum is already in that shorter form.